book/controller.rst

```diff
@@ -619,8 +619,7 @@ session.
 Flash Messages
 ~~~~~~~~~~~~~~
 
-You can also store small messages that will be stored on the user's session
-for exactly one additional request. This is useful when processing a form:
+You can also store small messages that will be stored on the user's session. This is useful when processing a form:
```

Comment:
Can you please revert this change? We wrap lines after the first word that crosses the 72nd character mark.